### PR TITLE
Postman examples for Card API

### PR DESCRIPTION
**Pull request by nilssonemma**

### CODE REVIEW

**Review by marcushanses — APPROVED** [review]

File: docs/.vuepress/public/assets/scripts/card-api-collection.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{baseUrl}}/card/{{token}}",
+					"host": [
+						"{{baseUrl}}"
```
Comment (Magnus93) [inline docs/.vuepress/public/assets/scripts/card-api-collection.json:78]: The baseUrl can be set for all requests

File: docs/merchant/card-api/change.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Change
+
+To update information of a card, send a PUT request to the card endpoint with the card token in the url. The body of the request should be a [Card Creatable](../../integrate/acquiring/reference.html#creatable-5). The response will be the updated card token.
+
+#### Request
+``` {1}
+PUT /card/<cardToken>
```
Comment (Magnus93) [inline docs/merchant/card-api/change.md:7]: `v1/card/card_token`

File: docs/merchant/card-api/fetch.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Fetch
+
+To fetch information about a card, make a GET call to the card endpoint with the card token included in the url. 
+
+#### Request
+``` {1}
+GET /card/<cardToken>
```
Comment (Magnus93) [inline docs/merchant/card-api/fetch.md:7]: `v1/card/<card_token>`

File: docs/.vuepress/public/assets/scripts/card-api-collection.json (hunk 1 above)
Comment (Magnus93) [inline docs/.vuepress/public/assets/scripts/card-api-collection.json:78]: The baseUrl can be set for all requests

File: docs/merchant/card-api/fetch.md (hunk 3 above)
Comment (Magnus93) [inline docs/merchant/card-api/fetch.md:7]: `v1/card/<card_token>` ?

File: docs/merchant/card-api/change.md (hunk 2 above)
Comment (Magnus93) [inline docs/merchant/card-api/change.md:7]: `v1/card/<card_token>` ?

**Review by Magnus93 — CHANGES_REQUESTED** [review]

File: docs/.vuepress/config.js [hunk 4]
```diff
@@ -166,6 +166,9 @@ function getMerchantSidebar() {
       collapsable: false,
       children: [
```
Comment (Magnus93) [inline docs/.vuepress/config.js:167]: Also add `"Card API"` to `nav` in config file.

**Review by Magnus93 — CHANGES_REQUESTED** [review]

**Review by Magnus93 — APPROVED** [review]

**Pull request merged**